Authorization Headers

phgoncalves opened the pull request:
## General changes:
 * Removed access token path param
 * Added authorization header

micheloliveiramarciano committed:
Update Token

micheloliveiramarciano committed:
Update maven

micheloliveiramarciano committed:
Update Token

Commit:
Merge from origin master

Commit:
Remove access token from URI

- Remove access token from URI
- Add access token on header

Commit:
Update MPRestClient.java

- Update authorization logic

Commit:
Delete launch.json

angrifel commented on src/main/java/com/mercadopago/net/MPRestClient.java:
```diff
@@ -192,14 +189,25 @@ public MPApiResponse executeRequest(HttpMethod httpMethod, String uri, PayloadTy
+            return MercadoPago.SDK.getAccessToken();
+
+        return SDK.getAccessToken();
+    }
```
would this achieve the goal?
```java 
    private String getAccessToken(MPRequestOptions requestOptions) throws MPException {
        if (requestOptions.getAccessToken() != null && requestOptions.getAccessToken().isEmpty())
            return requestOptions.getAccessToken();

        return SDK.getAccessToken();
    }
```

phgoncalves replied:
@angrifel
Yes, it would.
Thanks!

Commit:
Add marketplace token validation

- Add marketplace token validation
- Add oauth URL validation
- Remove unnecessary condition

Commit:
Update MPRestClient.java

Commit:
Merge branch 'develop' into enhancement/access-token-header

Commit:
Update MPBaseMultipleParamsTest.java

Remove access token path param

Commit:
Update MercadoPagoTest.java
